dev_db/creator.py

```diff
@@ -36,25 +38,23 @@ def get_models(self):
         excluded = self.get_excluded_models()
         all_models = self.get_all_models()
         valid_models = list(full_required)
-        
 
         for m, n in all_models:
             # skip the ones already in full required
             if m in valid_models:
                 continue
-            table = m._meta.db_table
-            # handle the excludes
-            valid = True
-            for e in excluded:
-                if e in table:
-                    valid = False
-            if getattr(m._meta, 'proxy'):
-                valid = False
-            if valid:
-                valid_models.append(m)
-                
+
+            if all(map(lambda x: x not in m._meta.db_table, excluded)) \
+                    and not getattr(m._meta, 'proxy'):
```

Comment:
We can remove `getattr` here. `proxy` is always set, and `getattr` raises `AttributeError` if the attr doesn't exist anyway.

Also not super keen on backslashes for breaking lines. We should probably add black to this to avoid bikeshedding though.

How about this?

```suggestion
            if (
                all(map(lambda x: x not in m._meta.db_table, excluded))
                and m._meta.proxy is False
            ):
                try:
```